ci/ci/ci.py

```diff
@@ -121,9 +121,7 @@
             status = await pr.batch.status()
             for j in status['jobs']:
                 if 'duration' in j and j['duration'] is not None:
-                    duration = Job.duration(j)
-                    if duration is not None:
-                        j['duration'] = humanize.naturaldelta(datetime.timedelta(seconds=duration))
+                    j['duration'] = humanize.naturaldelta(datetime.timedelta(seconds=sum(j['duration'])))
```

Comment:
what is the type of `j['duration']`?

Reply:
it should be a string I think from the docs

Reply:
oh, you meant the input. It should be an array of durations. One for setup, cleanup, and main.